conda_build/skeletons/cran.py

```diff
@@ -1165,6 +1179,29 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
         if 'R' not in dep_dict:
             dep_dict['R'] = ''
 
+        os_type = cran_package.get("OS_type", '')
+        if os_type != 'unix' and os_type != 'windows' and os_type != '':
+            print("Unknown OS_type: {} in CRAN package".format(os_type))
+            os_type = ''
+        if os_type == 'unix':
+            d['skip_os'] = 'skip: True  # [not unix]'
+            d["noarch_generic"] = ""
+        if os_type == 'windows':
+            d['skip_os'] = 'skip: True  # [not win]'
+            d["noarch_generic"] = ""
+        if os_type == '':
+            d['skip_os'] = '# no skip'
+        skips = ''
+        # print("seek for {} in blg_mapdeps".format(package))
+        skips = get_for_cran_skip(glb_mapdeps, package, '')
+        for n in dep_dict:
+            skips = get_for_cran_skip(glb_mapdeps, n, skips)
+        if skips != '':
+            skips.strip()
```

Comment:
I think you forgot to reassign this to ``skips`` maybe? or maybe is it a leftover?
It is because ``skips.strip()`` return a new string and ``skips`` will remain the same. Looking for the logic, maybe that can be something like
```python
if skips.strip():
    skips = "  # [" + skips + "]"
    d["skip_os"] = "skip: True" + skips
```

Reply:
Yes, thanks. The intention is here to have whitespace as indicator for "all".  But well, we could introduce here also the special value "all" instead to indicate that
I addressed it in current logic